src/persist-client/src/write.rs

```diff
@@ -966,4 +968,49 @@ mod tests {
         );
         assert_eq!(write.upper(), &Antichain::from_elem(7));
     }
+
+    #[mz_ore::test(tokio::test)]
+    #[cfg_attr(miri, ignore)] // unsupported operation: returning ready events from epoll_wait is not yet implemented
+    async fn fetch_recent_upper_linearized() {
```

Comment:
This is maybe not a super robust, but if it fails then something is definitely broken.

Reply:
Done